job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContextSpec.groovy

```diff
@@ -1488,4 +1488,39 @@ class WrapperContextSpec extends Specification {
         'key1' | 'key2' | 'key3'  | null   | 'key1'      | 'key2'      | 'key3'      | ''
         'key1' | 'key2' | 'key3'  | 'key4' | 'key1'      | 'key2'      | 'key3'      | 'key4'
     }
+
+    def 'call versionNumber'() {
```

Comment:
Sorry about this, I didn't review your test case, would you mind adding another TestCase for call versionNumber with all options'